config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml

```diff
@@ -197,7 +221,7 @@ presubmits:
             memory: "9Gi"
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
-      testgrid-tab-name: pull-conformance-v1alpha3
```

Comment:
I didn't check but if anything else references v1alpha3 in this file (in test descriptions, etc.) it needs to be changed to v1alpha4